pkg/server/chaosd/network.go

```diff
@@ -380,22 +389,20 @@ func (networkAttack) Recover(exp core.Experiment, env Environment) error {
 	case core.NetworkPortOccupiedAction:
 		return env.Chaos.recoverPortOccupied(attack, env.AttackUid)
 	case core.NetworkDelayAction, core.NetworkLossAction, core.NetworkCorruptAction, core.NetworkDuplicateAction, core.NetworkPartitionAction, core.NetworkBandwidthAction:
-		if attack.NeedApplyIPSet() {
-			if err := env.Chaos.recoverIPSet(env.AttackUid); err != nil {
-				return perrors.WithStack(err)
-			}
+		// `chaosdaemon.DeamonServer.SetTcs()` may build new iptables which will not be recorded in DB,
```

Comment:
We should update this comment.

Reply:
I don't think this comment should be updated, the filter iptables still not be stored in the DB

Reply:
Yes, but here we recover all because chaos mesh will apply all network actions when we apply a new one.

Reply:
I agree